client/lib/signup/site-type.js

```diff
@@ -4,14 +4,19 @@
 import i18n from 'i18n-calypso';
 import { find, get } from 'lodash';
 
+/**
+ * Internal Dependencies
+ */
+import config from '@automattic/calypso-config';
+
 /**
  * Internal dependencies
  */
 
 const getSiteTypePropertyDefaults = ( propertyKey ) =>
 	get(
 		{
-			theme: 'pub/hever',
+			theme: config.isEnabled( 'gutenboarding/site-editor' ) ? 'pub/seedlet-blocks' : 'pub/hever',
```

Comment:
Should we use a different flag?

Reply:
Did you have any flags in mind? The site editor flag seems reasonable to me.

Reply:
No other flags in mind. I'm just not sure if it's appropriate to use `gutenboarding/*` flag in here. I feel like gutenboarding flags belong to gutenboarding 😄 So we could have something like `signup/site-editor` or `onboarding/site-editor`? I'm not sure.

Reply:
I think `signup/*` is more appropriate here.
Although, please notice that we already have a `signup/full-site-editing` referring to the Legacy FSE!
We should probably be very clear with this new flag, and go with something very different like `signup/core-site-editor`.